src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/UserAccountManager.java

```diff
@@ -1999,4 +1999,49 @@ public Map<SchoolInfoStatus, Long> getSchoolInfoStats() throws SegueDatabaseExce
   public Long getNumberOfAnonymousUsers() throws SegueDatabaseException {
     return temporaryUserCache.getCountOfAnonymousUsers();
   }
+
+  public RegisteredUserDTO updateTeacherPendingFlag(final Long userId, boolean newFlagValue)
+      throws SegueDatabaseException, NoUserException {
+    RegisteredUser user = findUserById(userId);
+    if (user == null) {
+      throw new NoUserException("No user found with this ID.");
+    }
+    user.setTeacherPending(newFlagValue);
+    RegisteredUser updatedUser = database.createOrUpdateUser(user);
+    return dtoMapper.map(updatedUser, RegisteredUserDTO.class);
+  }
+
+  public void sendRoleChangeRequestEmail(final HttpServletRequest request, final RegisteredUserDTO currentUser,
+                                         final Role requestedRole, final Map<String, String> requestDetails)
+      throws SegueDatabaseException, ContentManagerException {
+    String roleName = requestedRole.toString();
+    String userSchool = Objects.requireNonNullElse(currentUser.getSchoolId(), currentUser.getSchoolOther());
+    String verificationDetails = requestDetails.get("verificationDetails");
+    String otherInformation = Objects.requireNonNullElse(requestDetails.get("otherInformation"), "");
+    String emailSubject = String.format("%s Account Request", roleName);
+    String emailMessage = String.format(
+        "Hello,\n<br>\n<br>"
+            + "Please could you convert my Isaac account into a teacher account.\n<br>\n<br>"
+            + "My school is: %s\n<br>"
+            + "A link to my school website with a staff list showing my name and email"
+            + " (or a phone number to contact the school) is: %s\n<br>\n<br>\n<br>"
+            + "Any other information: %s\n<br>\n<br>"
```

Comment:
Small one, but given that `otherInformation` is optional, maybe we can make the email a bit nicer by simply removing this line if there is no `otherInformation`. If it's not too messy. Otherwise, ignore this suggestion.